espnet2/asr/frontend/adapters_utils/README.md

```diff
@@ -0,0 +1,43 @@
+## Use adapters for ASR in ESPnet2
```

Comment:
Can you also add something about adapter in the espnet top level readme and tutorial doc as well?

Reply:
Sure thing, I would add a section in the main README

Reply:
- section added in top level README - ASR 
- [tutorial docs](https://espnet.github.io/espnet/tutorial.html#transducer) seems to be a seperate website?

Reply:
Hi, where would be a good place to add in the tutorial docs? Right now I have the full tutorial for adapters in adapter_utils/README.md.

Reply:
How about here? https://github.com/espnet/espnet/blob/master/doc/espnet2_tutorial.md#usage-of-self-supervised-learning-representations-as-feature

Reply:
Hi just a quick reminder that the updated documentes have been updated to contain the documentation of adapter modules and how to use them. Could you give the PR in its current state a quick look?